secure-releases.md

```diff
@@ -0,0 +1,60 @@
+
+### Two-factor Authentication
+
+Two-factor authentication (2FA) means having something more than just your password to confirm you're you. The second factor can be a one-time code using an app like Google Authenticator, a hardware key (such as Yubikey or Titan), or a passkey using touch ID, face ID, or something else that's stored locally on your device. Note: do not *ever* use SMS for 2FA unless it's the only mechanism offered, since SMS is vulnerable to SIM swap and number forgery attacks.
+
+Although this extra authentication step can be a mild inconvenience, the benefit for security is overwhelmingly worth it. Both Github and npm will soon require 2FA for every user, so you might as well enable it now.
+
+### Preparing for Publish
+
+It can be easy to forget to run steps before publishing. If you have any build steps (transpiling with Babel or TypeScript, emitting types with TypeScript, etc) and/or validation steps (running tests, etc), make sure they're set up in the `scripts` section of `package.json` such that `npm run prepublish` or `npm run prepublishOnly` would invoke them - that way, running `npm publish` in the future will automatically run these steps.
+
+Publishing to npm can be done in one step (`npm publish`), or two (`npm pack` to generate a tarball, and then later, `npm publish path/to/tarball` to publish). Even if you publish separately with one step, it's a good idea to run `npm pack` in CI to ensure that the steps succeed, and that the tarball is somewhat reproducible. (TODO: do this in my packages' CI and link to it here as an example)
+
+### Publishing
+If you have only 1 person who can publish releases, then this step is optional. If you have 2 or more, this step is strongly recommended.
+
+Publish a packed tarball from CI, only upon an explicit request. This means, don't make a new release automatically, but make sure that a human has intentionally said "it's time to publish a new version, here's the version number it will use". This explicit request can be a pushed semver-versioned git tag, or an explicit user-run workflow - but does not include merging/committing to the default branch.
+
+Do not publish without using 2FA. Note that an npm automation token is a single factor, and does not qualify as 2FA. Since Github does not provide any functionality to allow you to input a 2FA code natively, you'll need to use something like [Wombat Dressing Room](https://opensource.googleblog.com/2020/01/wombat-dressing-room-npm-publication_10.html) which requires you to host a service on GCP ([example](https://github.com/mathiasbynens/String.prototype.endsWith/blob/main/.github/workflows/publish-on-tag.yml#L21-L22)), or [Step Security's Wait For Secrets](https://github.com/step-security/wait-for-secrets) ([example](https://github.com/jsx-eslint/eslint-plugin-react/blob/28f9a6cdf716bc34f824ea3a3988e68f3e54eb0f/.github/workflows/npm-publish.yml)) which requires you to trust Step Security's external service.
```

Comment:
Is there anything that the Foundation can do to help maintainers implement the automate 2FA more easily?

Reply:
Short of maintaining a service for it, or pressuring Github to support it natively, I'm not sure what anyone can do to make it easier.

Reply:
> Short of maintaining a service for it
Is that at all reasonable as a value add by the Foundation for OpenJS projects?

Reply:
Yes, but the costs might be quite large over time - Foundation budget would be better spent paying Github to implement it natively, i suspect.